File: output/copilot/c/results/q18-count-numbers-with-unique-digits.c
/** https://leetcode.com/problems/count-numbers-with-unique-digits/ */
//Given an integer n, return the count of all numbers with unique digits, x, where 0 <= x < 10n.



int countNumbersWithUniqueDigits(int n){
    if(n==0) return 1;
    int res=10;
    int base=9;
    for(int i=2;i<=n;i++){
        base*=10-i+1;
        res+=base;
    }
    return res;
}
    

File: output/copilot/javascript/results/q18-count-numbers-with-unique-digits.js
/** https://leetcode.com/problems/count-numbers-with-unique-digits/ */
//Given an integer n, return the count of all numbers with unique digits, x, where 0 <= x < 10n.

/**
 * @param {number} n
 * @return {number}
 */
 var countNumbersWithUniqueDigits = function(n) {
    if (n === 0) {
   return 1;
}
if (n === 1) {
   return 10;
}
let result = 10;
let num = 9;
let count = 9;
while (n > 1) {
   count *= num;
   result += count;
   num--;
   n--;
}
return result;
};
